<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>新建模型</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #f6f8fa, stop:1 #e0e4e8);
}

QLabel {
    color: #333333;
    font-family: &quot;Microsoft YaHei UI&quot;;
    font-size: 10pt;
}

QLabel#titleLabel {
    color: #1e3a8a;
    font-size: 14pt;
    font-weight: bold;
}

QLineEdit, QTextEdit {
    background-color: rgba(255, 255, 255, 200);
    border: 1px solid #d0d0d0;
    border-radius: 5px;
    padding: 4px 8px;
    color: #333333;
    font-family: &quot;Microsoft YaHei UI&quot;;
}

QLineEdit:hover, QTextEdit:hover {
    border: 1px solid #a0a0a0;
    background-color: rgba(255, 255, 255, 230);
}

QLineEdit:focus, QTextEdit:focus {
    border: 1px solid #2473bd;
    background-color: white;
}

QPushButton {
    background-color: #ffffff;
    border: 1px solid #e0e0e0;
    border-radius: 5px;
    padding: 6px 16px;
    color: #333333;
    font-family: &quot;Microsoft YaHei UI&quot;;
}

QPushButton:hover {
    background-color: #f0f7ff;
    border: 1px solid #2473bd;
}

QPushButton:pressed {
    background-color: #e0e0e0;
    border: 1px solid #006ad1;
}

QPushButton#newButton {
    background-color: #1e88e5;
    color: white;
    font-weight: bold;
}

QPushButton#newButton:hover {
    background-color: #0d6efd;
}

QPushButton#newButton:pressed {
    background-color: #0b5ed7;
}</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>420</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>创建新模型</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="infoFrame">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>65</y>
     <width>420</width>
     <height>210</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#infoFrame {
    background-color: rgba(255, 255, 255, 100);
    border-radius: 10px;
    border: 1px solid rgba(200, 200, 200, 100);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>名称：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>311</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>输入模型名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="description">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>描述：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="descriptionEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>311</width>
     <height>131</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>输入模型描述信息（可选）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>295</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>创建(C)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>295</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <zorder>infoFrame</zorder>
  <zorder>titleLabel</zorder>
  <zorder>name</zorder>
  <zorder>nameEdit</zorder>
  <zorder>description</zorder>
  <zorder>descriptionEdit</zorder>
  <zorder>newButton</zorder>
  <zorder>cancelButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
